syntax = "proto3";

package main;

option go_package = ".";

message ResultCode {
    int32 errcode = 1;
    bytes errmsg = 2;
}

message LoginRequest {
    bytes name = 1;
    bytes pwd = 2;
}

message LoginResponse {
    ResultCode result = 1;
    bool success = 2;
    string test_name_1 = 3;
    bytes test_name_4 = 4;
}

service UserServiceRpc {
    rpc Login(LoginRequest) returns(LoginResponse);
}